### PR TITLE
feat(webapp): improve UI from main page

### LINKED ISSUE #53: Improve UI for main page
Les't  impruve the view from the main page and style 

- [x] Sidebard 
- [x] Footer 
- [x] Navbar

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #53 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. click on income in the sidebar and see new features

#### CheckList
- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check

### CODE REVIEW

File: webapp/src/components/Footer/index.js [hunk 1]
```diff
@@ -15,26 +12,64 @@ const useStyles = makeStyles(styles)
...
-                    {link.text}
-                  </a>
-                }
-              />
-            </ListItem>
-          ))}
-        </List>
+      <Grid container spacing={3}>
+        <Grid item xs={12} sm={4}>
+          <Box className={classes.footerInformation}>
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:19]: should we be using divs instead of box here?

File: webapp/src/language/es.json [hunk 2]
```diff
@@ -28,7 +28,8 @@
     "welcomeMessage": "El estándar de contabilidad de Eden es un mecanismo de informes destinado a proporcionar a la comunidad de Eden y EOS información sobre cómo se asignan los fondos de Eden. Los delegados electos pueden adherirse al estándar y reportar fácilmente sus gastos de Eden utilizando un formato específico para el campo de memo de la acción de transferencia de token EOS."
   },
   "incomeRoute": {
-    "title": "Ingresos por delegados",
+    "title": "Ingresos",
+    "subTitle": "Ingresos por delegados",
```
Comment (xavier506) [inline webapp/src/language/es.json:32]: Should it be: Ingresos por delegado (singluar). ?

File: webapp/src/components/Footer/index.js [hunk 3]
```diff
@@ -1,10 +1,7 @@
 import React, { memo } from 'react'
-import Grid from '@mui/material/Grid'
-import List from '@mui/material/List'
-import ListItemText from '@mui/material/ListItemText'
-import ListItem from '@mui/material/ListItem'
-import Box from '@mui/material/Box'
+import { Grid, Box } from '@mui/material'
```
Comment (tetogomez) [inline webapp/src/components/Footer/index.js:2]: according MUI docs this kind of import components `import Box from '@mui/material/Box'` is better than `import { Grid, Box } from '@mui/material'`
Please import de component directly.

Actually we are avoiding use Box and Grid components in order to keep up the performance

File: webapp/src/components/Footer/index.js [hunk 4]
```diff
@@ -15,26 +12,64 @@ const useStyles = makeStyles(styles)
...
+              <a href="https://github.com/eoscostarica/eden-accounting-standard/tags">
+                <Box display="flex">
+                  <GitMergeIcon /> Version {mainConfig.appVersion}
+                </Box>
+              </a>
+            </Box>
+          </Box>
+        </Grid>
       </Grid>
     </Box>
```
Comment (tetogomez) [inline webapp/src/components/Footer/index.js:74]: Refactor this component and remove Box and Grid

File: webapp/src/components/Header/index.js [hunk 5]
```diff
@@ -189,74 +188,86 @@ const Header = memo(({ onDrawerToggle }) => {
...
+          </Box>
+          <Box className={classes.mobileSection}>
+            <IconButton
+              aria-label="show more"
+              aria-haspopup="true"
+              onClick={handleOpenMenu}
+            >
+              <MoreIcon />
+            </IconButton>
+          </Box>
```
Comment (tetogomez) [inline webapp/src/components/Header/index.js:237]: same in this file

**Review by tetogomez — CHANGES_REQUESTED** [review]
Please Remove Box and Grid from the whole PR

**Review by xavier506 — COMMENTED** [review]
are you sure you want to delete 
`hapi/src/services/hyperion/updaters/genesis-fundtransfer.updater.js`
and 
`hasura/seeds/default/1661533517069_initialEdenHistoricElectionSeed.sql`

?

**Comment by JAMares** [comment]
The files deleted are by my recommendation, these are unused files that deleted on my PR, but both there were when created this branch

File: webapp/src/components/Footer/index.js [hunk 6]
```diff
@@ -15,26 +12,64 @@ const useStyles = makeStyles(styles)
...
-            </ListItem>
-          ))}
-        </List>
+      <Grid container spacing={3}>
+        <Grid item xs={12} sm={4}>
+          <Box className={classes.footerInformation}>
+            <img
+              width="140"
+              height="100"
+              src="https://guias.eoscostarica.io/assets/images/eoscostarica-logo-790106817dcb2928ca190a0723d48662.png"
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:23]: this image should be stored in the `webapp/static/` folder

File: webapp/src/components/Footer/index.js [hunk 7]
```diff
@@ -15,26 +12,64 @@ const useStyles = makeStyles(styles)
...
+            <Box
+              display="flex"
+              alignItems="center"
+              flexDirection="column"
+              mt={2}
+            >
+              <a href="https://edeneos.org/">Website EOS Costa Rica</a>
+              <a href="https://t.me/eoscr">Telegram Channel</a>
+              <a href="https://www.linkedin.com/company/eoscostarica/mycompany/">
+                Linkedin
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:43]: LinkedIn (capital I)

File: webapp/src/components/Footer/index.js [hunk 8]
```diff
@@ -15,26 +12,64 @@ const useStyles = makeStyles(styles)
...
+        <Grid item xs={12} sm={4}>
+          <Box className={classes.footerInformation}>
+            EOS Costa Rica
+            <Box
+              display="flex"
+              alignItems="center"
+              flexDirection="column"
+              mt={2}
+            >
+              <a href="https://edeneos.org/">Website EOS Costa Rica</a>
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:40]: this is not the eos costa rica website URL

File: webapp/src/components/Footer/index.js [hunk 9]
```diff
@@ -15,26 +12,64 @@ const useStyles = makeStyles(styles)
...
+              <a href="https://t.me/eoscr">Telegram Channel</a>
+              <a href="https://www.linkedin.com/company/eoscostarica/mycompany/">
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:42]: https://www.linkedin.com/company/eoscostarica

File: webapp/src/components/Header/index.js [hunk 10]
```diff
@@ -189,74 +188,86 @@ const Header = memo(({ onDrawerToggle }) => {
...
+        <Toolbar className={classes.toolbar}>
+          <Hidden mdUp>
+            <IconButton aria-label="Open drawer" onClick={onDrawerToggle}>
+              <MenuIcon />
+            </IconButton>
+          </Hidden>
+          <Box className={classes.typography}>
+            <img
+              height={47}
+              src={process.env.PUBLIC_URL + '/EOS-Spend-Explorer.png'}
```
Comment (xavier506) [inline webapp/src/components/Header/index.js:207]: we should keep images in `webapp/static/images` folder

File: webapp/src/routes/IncomeReport/index.js [hunk 11]
```diff
@@ -81,70 +86,99 @@ const IncomeReport = () => {
...
+        <Container>
+          <Box textAlign="justify" marginTop={2}>
+            <Typography variant="span" className={classes.subTitle}>
+              {t('subTitle')}
+            </Typography>
+            <br />
+            <Typography variant="span">
+              {t('textInformation')}
+              <br />
+              Next disbursement date: June 8, 2022
```
Comment (xavier506) [inline webapp/src/routes/IncomeReport/index.js:99]: This date needs to be sourced from the chain tables. we can create a new issue for this

File: webapp/src/routes/IncomeReport/IncomeChart.js [hunk 12]
```diff
@@ -13,17 +14,22 @@ import {
...
 
+import styles from './styles'
+
+const useStyles = makeStyles(styles)
+
 const RenderChartLegend = ({ data }) => {
+  const classes = useStyles()
   return (
-    <ul style={{ listStyle: 'none' }}>
+    <ul style={{ listStyle: 'none', overflowY: 'auto', maxHeight: '400px' }}>
```
Comment (tetogomez) [inline webapp/src/routes/IncomeReport/IncomeChart.js:24]: Aviod to use inline style, create a class and use classes const

**Review by tetogomez — CHANGES_REQUESTED** [review]
Just one comment more

File: webapp/src/layouts/Dashboard/index.js [hunk 13]
```diff
@@ -1,15 +1,15 @@
...
 import { makeStyles } from '@mui/styles'
 
 import Sidebar from '../../components/Sidebar'
 import Header from '../../components/Header'
-import Footer from '../../components/Footer'
 import Message from '../../components/Message'
+import Footer from '../../components/Footer'
 
 import styles from './styles'
+import { Toolbar } from '@mui/material'
```
Comment (leisterfrancisco) [inline webapp/src/layouts/Dashboard/index.js:12]: please, import it in top of this file.

import order:
1. Third-party libraries
2. Project import
3. Folder import

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**